core/src/test/scala/unit/kafka/server/ApiVersionsRequestTest.scala

```diff
@@ -83,7 +83,7 @@ object ApiVersionsRequestTest {
 class ApiVersionsRequestTest(cluster: ClusterInstance) extends AbstractApiVersionsRequestTest(cluster) {
 
   @ClusterTemplate("testApiVersionsRequestTemplate")
-  @ClusterTest(types = Array(Type.KRAFT, Type.CO_KRAFT), metadataVersion = MetadataVersion.IBP_4_0_IV0, serverProperties = Array(
+  @ClusterTest(types = Array(Type.KRAFT, Type.CO_KRAFT), metadataVersion = MetadataVersion.IBP_3_8_IV0, serverProperties = Array(
```

Comment:
We cannot use `latestTesting` here since we have to use an enum, not a function. I think this is a big weakness of `@ClusterTemplate`, but we're stuck with it.

However, I can leave out the `metadataVersion` annotation and it will default to the latest testing.